core/src/services/mod.rs

```diff
@@ -334,6 +334,20 @@ pub use upyun::Upyun;
 #[cfg(feature = "services-upyun")]
 pub use upyun::UpyunConfig;
 
+#[cfg(feature = "services-chainsafe")]
+mod chainsafe;
```

Comment:
These changes should not be included in this PR. Perhaps you need to rebase your branch?

Reply:
I try to learn how to use Jetbrains's IDE instead of command line. It seems ok now.

Reply:
You could try to pull the origin into your branch using rebase. Just right click the origin main branch, you'll find this menu item.